tests/testsuite/login.rs

```diff
@@ -185,4 +187,18 @@ fn registry_credentials() {
 
     // Also ensure that we get the new token for the registry
     assert!(check_token(TOKEN, Some(reg)));
+
+    let reg2 = "alternative2";
+    cargo_process("login --registry")
+        .arg(reg2)
+        .arg(TOKEN2)
+        .arg("-Zunstable-options")
+        .masquerade_as_nightly_cargo()
```

Comment:
These two lines can be removed, I think we just forgot to remove them when stabilized. And remove the same 2 lines in the call above.